include/v810_mem.h

```diff
@@ -5,27 +5,31 @@
 
 // Memory Structure for the VIP Reg (Could have done it with an array
 // but this is pretier...)
-typedef struct {
-    HWORD INTPND;
-    HWORD INTENB;
-    HWORD INTCLR;
-    HWORD DPSTTS;
-    HWORD DPCTRL;
-    HWORD BRTA;
-    HWORD BRTB;
-    HWORD BRTC;
-    HWORD REST;
-    HWORD FRMCYC;
-    HWORD CTA;
-    HWORD XPSTTS;
-    HWORD XPCTRL;
-    HWORD tFrameBuffer;   //not publicly visible
-    HWORD tFrame;
-    HWORD VER;
-    HWORD SPT[4];
-    HWORD GPLT[4];
-    HWORD JPLT[4];
-    HWORD BKCOL;
+typedef union{
+    struct {
+        HWORD INTPND;
+        HWORD INTENB;
+        HWORD INTCLR;
+        HWORD DPSTTS;
+        HWORD DPCTRL;
+        HWORD BRTA;
+        HWORD BRTB;
+        HWORD BRTC;
+        HWORD REST;
+        HWORD FRMCYC;
+        HWORD CTA;
+        HWORD XPSTTS;
+        HWORD XPCTRL;
+        HWORD tFrameBuffer;   //not publicly visible
+        HWORD tFrame;
+        HWORD VER;
+        HWORD SPT[4];
+        HWORD GPLT[4];
+        HWORD JPLT[4];
+        HWORD BKCOL;
+    };
+
+    HWORD V810_VIPREGARR[20];
```

Comment:
Is this needed for anything? If it's intended to treat the whole struct as one array, it's incorrect, as while there are 20 variables, some of them are arrays.

Reply:
No, it's not needed, it was intended for that, but I wasn't able to make it work and forgot to rollback this specific part of it, shoul be reverted in [50c4918](https://github.com/skyfloogle/red-viper/pull/59/commits/50c4918ab30979da68f07b02d38827556a11cf80)